### DISCUSSION TITLE
posts/6066

### DISCUSSION THREAD

**Discussion by giscus** [Alanhays/Alanhays.github.io#3] [Announcements]
# posts/6066

# 题目 猿人学 2023 届第三题 点击跳转 # 0x01 降维打击 这里 if 都是多层嵌套的，非常影响阅读体验，直接一步将它变为一维结构吧！ 还原前先手动还原小部分代码，接着写反混淆脚本批量修改。 我们单看  if (o < 2)  这个 if 块，很明显 o < 1  其实就是 o==0  而 else 就是  o==1 , 以此类推。 12345678910111213141

https://alanhays.github.io/posts/6066.html

**Comment by Alanhays** [comment]
test
